frontend/src/components/transactions/ActionsList.tsx

```diff
@@ -164,9 +164,8 @@ export default class extends React.Component<Props> {
 
           .action-row-title {
             font-family: BentonSans;
-            font-size: 14px;
+            font-size: 1rem;
```

Comment:
Why this change? Makes the font quite a bit larger

Reply:
because this makes the logo and text not in the same baseline and could not center